PORTAL/jobs/_utils.py

```diff
@@ -232,12 +286,12 @@ def _normalize_columns(cls, specs, names, maxwidth):
             if i > 0:
                 size += len(sep)
             size += margin + columns[i].width + margin
-            if size > maxwidth:
+            if size > maxwidth_int:
                 # XXX Maybe drop other columns than just the tail?
                 # XXX Maybe combine some columns?
                 columns[i:] = []
-                if names:
-                    names[i:] = []
+                if names_list is not None:
```

Comment:
We want to skip the empty list too.

```suggestion
                if names_list:
```